apps/src/lab2/views/components/Instructions/ResourcePanel/styles.module.scss

```diff
@@ -214,7 +214,7 @@
 
 .navigationFooter {
   padding: 10px;
-  background-color: var(--background-neutral-secondary);
```

Comment:
It is part of this ticket to update the background color to have it stand out against feedback messages in the instructions panel... so that the button doesn't look like it is relating to any feedback or messaging that might appear directly above it.